framework_tvb/tvb/interfaces/web/controllers/spatial/local_connectivity_controller.py

```diff
@@ -95,7 +96,6 @@ def step_1(self, do_reset=0, **kwargs):
             if default_surface_index:
                 new_lconn.surface = uuid.UUID(default_surface_index.gid)
             else:
-                # TODO: ok to keep a default gid here?
                 new_lconn.surface = uuid.uuid4()
```

Comment:
This line makes not much sense `new_lconn.surface = uuid.uuid4()`. 
Why not put here `common.set_error_message("There ....` ?

Reply:
that line is used after in common.add2session(KEY_LCONN, new_lconn) which is used after in current_lconn, and is we don't have that we get a error when we do: configure_lcon_form.fill_from_trait(current_lconn) that redirects the user to the generic error page.